netlify.toml

```diff
@@ -131,1153 +115,1041 @@ to = "/account/my-organization/notifications/slack/slack-notifications"
 [[redirects]]
-  from = "/workflows/ios-specific-workflow-steps/distribution/appcenter-ios-distribution"
-  to = "/workflows/ios-specific-workflow-steps/appcenter-ios-distribution"
-  status = 301
+from = "/store-submit/google-play"
+to = "/publish-module/send-to-googleplay"
+status = 301
 
 [[redirects]]
-  from = "/workflows/ios-specific-workflow-steps/distribution/browserstack-app-automation"
-  to = "/workflows/ios-specific-workflow-steps/browserstack-app-automation"
-  status = 301
+from = "/store-submit/huawei-app-gallery"
+to = "/publish-module/send-to-huawei"
+status = 301
 
 [[redirects]]
-  from = "/workflows/ios-specific-workflow-steps/distribution/install-certificates-provisions"
-  to = "/workflows/ios-specific-workflow-steps/install-certificates-provisions"
-  status = 301
+from = "/store-submit/apple-app-store"
+to = "/publish-module/send-to-appstore"
+status = 301
 
 [[redirects]]
-  from = "/workflows/android-specific-workflow-steps/increment-build-and-version-number"
-  to = "/workflows/android-specific-workflow-steps/build-and-test/increment-build-and-version-number"
-  status = 301
+from = "/publish-module/send-to-appstore"
+to = "/publish-integrations/ios-publish-integrations/send-to-app-store"
+status = 301
 
-# Swagger wrong (broken) URL on publish module API docs
 [[redirects]]
-  from = "/store"
-  to = "/publish-module"
-  status = 301
+from = "/publish-module/send-to-googleplay"
+to = "/publish-integrations/android-publish-integrations/publish-to-google-play"
+status = 301
 
 [[redirects]]
-  from = "/workflows/android-specific-workflow-steps/build-and-test/android-unit-tests"
-  to = "/workflows/android-specific-workflow-steps/android-unit-tests"
-  status = 301
+from = "/publish-integrations/android-publish-integrations/send-to-googleplay"
+to = "/publish-integrations/android-publish-integrations/publish-to-google-play"
+status = 301
 
 [[redirects]]
-  from = "/workflows/android-specific-workflow-steps/distribution/appdome-build-to-secure-for-android"
-  to = "/workflows/android-specific-workflow-steps/appdome-build-to-secure-for-android"
-  status = 301
+from = "/publish-module/send-to-huawei"
+to = "/publish-integrations/android-publish-integrations/publish-to-huawei-appgallery"
+status = 301
 
 [[redirects]]
-  from = "/workflows/android-specific-workflow-steps/distribution/bundle-universal-apk"
-  to = "/workflows/android-specific-workflow-steps/bundle-universal-apk"
-  status = 301
+from = "/publish-integrations/android-publish-integrations/send-to-huawei"
+to = "/publish-integrations/android-publish-integrations/publish-to-huawei-appgallery"
+status = 301
 
 [[redirects]]
-  from = "/workflows/android-specific-workflow-steps/build-and-test/gradle-runner"
-  to = "/workflows/android-specific-workflow-steps/gradle-runner"
-  status = 301
+from = "/publish-module/send-to-googleplay#deleting-android-publish-profiles"
+to = "/publish-module/creating-publish-profiles/managing-publish-profiles#delete-publish-profile"
```

Comment:
This redirect does not work. Previously, I shared a long post on another PR.
- https://github.com/appcircleio/appcircle-docusaurus/pull/717#discussion_r1817995559

It should be worked on carefully. When you try the `from` here, you will see the `to` is not the result.